tests/test_openjdk.py

```diff
@@ -280,3 +297,24 @@ def test_jdk_cassandra(container_per_test):
     container_per_test.connection.check_output(
         f"cd /tmp/{cassandra_base}/tools/bin/ && ./cassandra-stress write n=1 && ./cassandra-stress read n=1",
     )
+
+
+@pytest.mark.parametrize(
+    "container",
+    FIPS_OPENJDK_IMAGES,
+    indirect=True,
+)
+def test_openjdk_sec_providers(container: ContainerData) -> None:
+    """
+    Verifies that the primary security provider in FIPS-enabled OpenJDK containers is `SunPKCS11-NSS-FIPS`.
+    The test uses Java scripts to list and validate security providers, ensuring FIPS compliance.
+    """
+    c = container.connection
+
+    c.check_output("git clone -q https://github.com/ecki/JavaCryptoTest")
```

Comment:
Relying on foreign repositories not under our control is not good. Can we not copy the repo to this repository as well?

Reply:
Agree. Also in line with @ricardobranco777 comment. I'll set the PR to draft to perform the changes.

Reply:
We do that for node & rust tests too, so it's not a huge problem in principle. I'm more worried that this project is effectively dead now: https://github.com/ecki/JavaCryptoTest/graphs/contributors

Reply:
Changed to a local `tar.gz`

Reply:
:heavy_check_mark: